src/index.html

```diff
@@ -11,7 +11,7 @@
     <div data-qa="big" class="container">
       <img
         src="images/Search.svg"
-        alt="#"
+        alt="img"
```

Comment:
alt should not contain words like image, img, icon and so on

sreenreaders read it and they already know this is an image